rollup.config.js

```diff
@@ -87,18 +87,71 @@ function createCommonJSConfig(input, output) {
   }
 }
 
+function createUMDConfig(input, output) {
+  return {
+    input,
+    output: {
+      file: output,
+      format: 'umd',
+      exports: 'named',
+      name: 'zustand',
+      esModule: true,
+    },
+    external,
+    plugins: [
+      alias({
+        entries: {
+          './vanilla': 'valtio/vanilla',
+          '../vanilla': 'valtio/vanilla',
```

Comment:
So, this is the trick.

Reply:
Yeah! I'm following the same config for alias